src/Microsoft.Extensions.Logging.Testing/Microsoft.Extensions.Logging.Testing.csproj

```diff
@@ -15,6 +15,7 @@
   </ItemGroup>
 
   <ItemGroup>
```

Comment:
Hmm, you know, I didn't realize that this project already was xUnit-specific, which is very odd to me. It's so far just because of LogValuesAssert, which throws an xUnit-specific exception.

Is there any reason for this library to be tied to xUnit? The LogValuesAssert case could just change to throw a regular exception (not as nice as xUnit's exception, but it'll work just fine).

Then xUnit-specific stuff could go into a new library.

Thoughts?